Iteam1337/iteamse-2018
# fix: helmet memory leak

**Ticket:** [#74](https://trello.com/c/ZZeUMHc7/74-fixa-react-helmet-minnesl%C3%A4ckan)
**Intent/Purpose:** 

To fix the memory leak in React Helmet.

**How to test (optional):**

there's a branch at iteam-cms with a cacheproxy and then run a script similar to
```
const http = require('http')

const REQUESTS_PER_SECOND = 10
const HOST = 'http://localhost:3000'

const get = () => http.get(HOST)

setInterval(get, 1000 / REQUESTS_PER_SECOND)
```

and observe the memory usage on the iteamse node-client.

* [ ] `npm run cypress` (optional)
